CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/SphereGraphBuilderPlugin.java

```diff
@@ -346,7 +346,12 @@ public void edit(final GraphWriteMethods graph, final PluginInteraction interact
                 graph.setStringValue(vxForegroundIconAttr, vxId, DEFAULT_ICON);
                 graph.setStringValue(vxDecoratorAttr, vxId, null);
             }
-            graph.setStringValue(vxBackgroundIconAttr, vxId, useRandomIcons ? (random.nextBoolean() ? "Background.Flat Square" : "Background.Flat Circle") : "Background.Flat Circle");
+
+            if (useRandomIcons) {
+                graph.setStringValue(vxBackgroundIconAttr, vxId, (random.nextBoolean() ? "Background.Flat Square" : "Background.Flat Circle"));
+            } else {
+                graph.setStringValue(vxBackgroundIconAttr, vxId, "Background.Flat Circle");
+            }
```

Comment:
could this be merged so that it uses the ternary condition but the initial condition is `useRandomIcons && random.nextBoolean()`